docs/UserGuide.md

```diff
@@ -128,24 +128,22 @@ Examples:
 * `editc 2 n/Betsy Crower t/` Edits the name of the 2nd contact to be `Betsy Crower` and clears all existing tags.
 
 
-### Locating contacts by name: `find`
+### Locating contacts by name: `findc`
 
 Finds contact whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `findc [n/NAME]... [t/TAGS]...`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
+* The search is case-insensitive. e.g. `hans` will match `Hans`
+* The order of the keywords does not matter. e.g. `n/Hans n/Bo` will match `n/Bo n/Hans`
+* Only the name and tag are searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
 * Contacts matching at least one keyword will be returned.
```

Comment:
The example below says "`find n/alex t/friends` returns `Alex Yeoh` if the Contact contains `friends` tag" (which is correct), so a user might find it confusing because the description on this line and the example seem to contradict each other.